.dockerignore

```diff
@@ -1,5 +1,6 @@
 *
 !src/
+!migrations/
```

Comment:
Oh, the migrations folder is not tracked?

Reply:
It's tracked in git, and this ensures it's also included in the Docker image.